src/app/content/components/Page/PageContent.tsx

```diff
@@ -60,6 +66,35 @@ export default styled(MainContent)`
     .highlight.${style.label} {
       background-color: ${style.passive};
 
+      &.block {
+        display: block;
+
+        &:after {
+          position: absolute;
+          z-index: -1;
+          content: "";
+          display: block;
+          width: calc(100% + 2rem);
+          height: calc(100% + 2rem);
+          top: -1rem;
+          left: -1rem;
```

Comment:
you could also set bottom and right to `-1` and you wouldn't need the calc in the width/height

Reply:
done